c10/util/typeid.h

```diff
@@ -623,8 +623,9 @@ CAFFE_DECLARE_PREALLOCATED_KNOWN_TYPE(
     26,
     detail::_guard_long_unique<std::vector<long>>)
 
-CAFFE_DECLARE_PREALLOCATED_KNOWN_TYPE(27, c10::qint8);
-CAFFE_DECLARE_PREALLOCATED_KNOWN_TYPE(28, _CaffeHighestPreallocatedTypeId)
-CAFFE_DECLARE_PREALLOCATED_KNOWN_TYPE(29, float*)
-CAFFE_DECLARE_PREALLOCATED_KNOWN_TYPE(30, at::Half*)
+CAFFE_DECLARE_PREALLOCATED_KNOWN_TYPE(27, float*)
```

Comment:
wait, are you allowed to change the mapping here?  This isn't serialized anywhere?

Reply:
I believe this is newly added

Reply:
Also they did it wrong by adding it after _CaffeHighestPreallocatedTypeId

Reply:
Yes, it's fine to change it - it's runtime-only mapping that doesn't get serialized